### PR TITLE
add new doc for multi job posts end point

### PR DESCRIPTION
**Pull request by capablemonkey**

### CODE REVIEW

File: source/includes/harvest/_job_posts.md [hunk 1]
```diff
@@ -250,3 +250,98 @@ questions | If present, will return an array of questions associated with this j
 
 <br>
 [See noteworthy response attributes.](#the-job-post-object)
+
+## Retrieve multiple job posts for a job
```
Comment (capablemonkey) [inline source/includes/harvest/_job_posts.md]: Let's say 'List job posts for a job' which is the more conventional way of saying this

File: source/includes/harvest/_job_posts.md [hunk 2]
```diff
@@ -250,3 +250,98 @@ questions | If present, will return an array of questions associated with this j
...
+        "label": "Last Name",
+        "type": "short_text",
+        "values": []
+      }
+    ]
+  }
+]
+```
+
+We recently updated our job posts modal to allow a single job to have multiple job posts.
```
Comment (capablemonkey) [inline source/includes/harvest/_job_posts.md]: Hmm, I don't think it is necessary to point out that this is a change.  We don't do this elsewhere, and also, at some point we'll have to remove it when it's no longer 'recent'  :)

File: source/includes/harvest/_job_posts.md [hunk 3]
```diff
@@ -250,3 +250,98 @@ questions | If present, will return an array of questions associated with this j
...
+
+Retrieve all the coresponding job posts for a given Job ID.
+
+
+### HTTP Request
+
+`GET https://harvest.greenhouse.io/v1/jobs/{id}/job_posts`
```
Comment (capablemonkey) [inline source/includes/harvest/_job_posts.md]: Do you think it'd make sense to rename the old endpoint now that there is a concept of multiple job posts for a single job?  

`Retrieve job post for job`

What does the older endpoint return if there are multiple job posts?  The "main" job post?  Or just one at random?  Could we call it something like 

`Retrieve main job post for job`
`Retrieve first job post for job`

Comment (capablemonkey) [inline source/includes/harvest/_job_posts.md]: Or, maybe we could keep it as is but distinguish the old one from the new by adding 

```
<aside class="warning">We recently introduced the ability to add multiple job posts for a single job, see <a href="#list-job-posts-for-job">here</a>.</aside>
```

**Comment by marunbai** [comment]
updated. second pass?

File: source/includes/harvest/_job_posts.md [hunk 4]
```diff
@@ -250,3 +252,96 @@ questions | If present, will return an array of questions associated with this j
 
 <br>
 [See noteworthy response attributes.](#the-job-post-object)
+
+## List multiple job posts for a job
```
Comment (capablemonkey) [inline source/includes/harvest/_job_posts.md]: Can we drop the word 'multiple' to keep the title short for the navigation bar on the left?  `

```
## List job posts for a job
```

**Comment by capablemonkey** [comment]
LGTM!

**Pull request merged**